### PR TITLE
Update `shell-session` default color

### PR DESCRIPTION
**Pull request by jorytindall**
### :pushpin: Summary

This is a draft PR to capture some of the different examples referenced in [these Figma mockups](https://www.figma.com/file/noyY6dUMDYjmySpHcMjhkN/branch/ORBrWlIRWeSs5SgDlSWBWp/HDS-Product---Components?type=design&node-id=52600%3A5383&mode=design&t=AbnZ8vAjm7hsyF5w-1) around syntax highlighting for shell-session examples and bash examples.

Can be trashed, this is meant to be more representative.

### :link: External links

Jira ticket: [HDS-2949](https://hashicorp.atlassian.net/browse/HDS-2949)
Figma file: [CodeBlock Color Updates](https://www.figma.com/file/noyY6dUMDYjmySpHcMjhkN/branch/ORBrWlIRWeSs5SgDlSWBWp/HDS-Product---Components?type=design&node-id=52600%3A5383&mode=design&t=AbnZ8vAjm7hsyF5w-1)

***

### 👀 Component checklist

- [x] Percy was checked for any visual regression
- [x] A11y tests have been run locally (`yarn test:a11y --filter="COMPONENT-NAME"`)
- [x] If documenting a new component, an acceptance test that includes the `a11yAudit` has been added
- [x] A changelog entry was added via [Changesets](https://github.com/changesets/changesets) if needed (see [templates here](https://github.com/hashicorp/design-system/blob/main/wiki/Website-Changelog.md#templates-for-npm-packages))

:speech_balloon: Please consider using [conventional comments](https://conventionalcomments.org/) when reviewing this PR.

[HDS-2949]: https://hashicorp.atlassian.net/browse/HDS-2949?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/components/tests/dummy/app/templates/components/code-block.hbs [hunk 1]
```diff
@@ -496,6 +501,95 @@ end'
     </SG.Item>
   </Shw::Grid>
 
+  <Shw::Divider @level="2" />
+
+  <Shw::Text::H3>Shell Session examples</Shw::Text::H3>
+
+  <Shw::Grid @columns={{2}} {{style gap="2rem"}} as |SG|>
+
```
Comment (KristinLBradley) [inline packages/components/tests/dummy/app/templates/components/code-block.hbs:509]:
```suggestion
```

Comment (KristinLBradley) [reply packages/components/tests/dummy/app/templates/components/code-block.hbs:509]: Removing blank line above indented code

**Review by didoo — APPROVED** [review]
👍

**Comment by KristinLBradley** [comment]
I commented within the [Figma file](https://www.figma.com/file/noyY6dUMDYjmySpHcMjhkN/branch/ORBrWlIRWeSs5SgDlSWBWp/HDS-Product---Components?type=design&node-id=52600-5383&mode=design&t=9i5e5Zad9QZbJljp-0) and within [Jory's Slack thread](https://hashicorp.slack.com/archives/C025N5V4PFZ/p1705614980398189?thread_ts=1705607082.518119&cid=C025N5V4PFZ)

**Review by KristinLBradley — APPROVED** [review]

**Pull request merged**